src/main/java/seedu/guest/model/guest/NumberOfGuests.java

```diff
@@ -0,0 +1,53 @@
+package seedu.guest.model.guest;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.guest.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the total number of guests a Guest has in the address book.
```

Comment:
Should this be guest book instead of address book?

Reply:
Yes it should. Fixed on the latest commit.